CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
 - On X11, add mappings for numpad comma, numpad enter, numlock and pause.
+- On Mac, reversed horizontal scrolling in `WindowEvent::MouseWheel` to match the direction of vertical scrolling.
```

Comment:
This is arguably a breaking change, perhaps mark it as such (using `**Breaking:**`) to be safe?